### PR TITLE
iOS Doc

### PR DESCRIPTION
**Pull request by tarruk**
The iOS documentation was added to explain how to implement FlowForms in iOS and how to use the provided tools within it

### CODE REVIEW

**Review by kstoletniy — APPROVED** [review]

File: docs/pages/android-utils/Binding.md [hunk 1]
```diff
@@ -57,11 +57,3 @@ private fun bindFields() {
...
 
 <div class="rs-row comment"> <i class="comment-icon fa-solid fa-circle-info"></i> <div class="comment">when interacting with a checkbox and using Two-way View binding at the same time, it becomes almost necessary to use a LiveData to get notified of the value changes, because Checkbox Views doesn't allow to have more than one onValueChange listener and doesn't provide any method to get the current listener.</div> </div>
-
-
-### Why do we need to pass lifecycleScope when using the bind extensions?
-
-The bind methods require to pass the `lifecycleScope` as argument because the validate functions in the FlowForm are `suspending functions`, hence they need to be called from a `coroutine`, and using the `lifecycleScope` gives us the benefit that anything we called will be immediately cancelled if the `lifecycleScope` is `cancelled/destroyed`. 
-
-In the case of binding a `LiveData` to a `FlowField`, we also need to pass a `LifecycleOwner` to be able to listen to the `LiveData` changes only when the fragment's/activity's lifecycle is on an active state (`started` or `resumed`). basically we call the `LiveData.observe(...)` method using the given `LifecycleOwner`.
-
```
Comment (hrodrick) [inline docs/pages/android-utils/Binding.md:67]: I think this shouldn't be removed, right?

File: docs/pages/iOS-utils/FlowForm-bindings.md [hunk 2]
```diff
@@ -0,0 +1,48 @@
+---
+layout: default
+title: FlowForms docs - Forms
+---
+
+### FlowForm bindings
+
+The FlowForms iOS library is designed to simplify the process of handling form data in your Swift code. 
+It provides a function that allows you to bind the publisher in your Swift code with the real-time status generated by the FlowForm.
```
Comment (hrodrick) [inline docs/pages/iOS-utils/FlowForm-bindings.md:9]:
```suggestion
It provides a function that allows you to bind a publisher in your Swift code with the real-time status generated by the FlowForm.
```

File: docs/pages/ios-quickstart.md [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+            SecureField("Password textfield", text: formState.password)
+            if let passwordErrorMessage = formState.passwordErrorMessage {
+                ErrorMessageView(message: passwordErrorMessage)
+            }
+        }
+    }
+}
+</code></pre>
+
+And that's one of the easiest forms we can create using **FlowForms**, we don't need to care about managing the field's validation triggering nor making any complex logic to enable or disable the continue button.
```
Comment (hrodrick) [inline docs/pages/ios-quickstart.md:164]: I think we are missing the part of the enabling/disabling of the button based on the form's status, what do you think about adding it?

File: docs/pages/kmp-quickstart.md [hunk 4]
```diff
@@ -73,33 +75,47 @@ class Bar {
...
 
-And that's one of the easiest forms we can create using **FlowForms** in KMP, we don't need to care about managing the field's status manually nor making any complex logic to enable or disable the continue button. 
 
-**FlowForms**'s full potential is better appreciated when making more complex forms, for an android usage example you can review the [android example app](https://github.com/rootstrap/FlowForms/tree/main/ExampleApp%20Android/src/main/java/com/rootstrap/flowforms/example) included the project. Which makes use of asynchronous validations and many other capabilities. You will see that the implementation steps doesn't change at all, except that it uses some specific android utilities in the UI layer. BTW, there is an example using Activity and another one using Fragment.
+And that's one of the easiest forms we can create using **FlowForms** in KMP, we don't need to care about managing the field's status manually nor making any complex logic to enable or disable the continue button.
+
+### How to use the library in a iOS app
```
Comment (hrodrick) [inline docs/pages/kmp-quickstart.md:104]:
```suggestion
### How to use the library in an iOS app
```

File: docs/pages/kmp-quickstart.md [hunk 5]
```diff
@@ -73,33 +75,47 @@ class Bar {
...
 
-And that's one of the easiest forms we can create using **FlowForms** in KMP, we don't need to care about managing the field's status manually nor making any complex logic to enable or disable the continue button. 
 
-**FlowForms**'s full potential is better appreciated when making more complex forms, for an android usage example you can review the [android example app](https://github.com/rootstrap/FlowForms/tree/main/ExampleApp%20Android/src/main/java/com/rootstrap/flowforms/example) included the project. Which makes use of asynchronous validations and many other capabilities. You will see that the implementation steps doesn't change at all, except that it uses some specific android utilities in the UI layer. BTW, there is an example using Activity and another one using Fragment.
+And that's one of the easiest forms we can create using **FlowForms** in KMP, we don't need to care about managing the field's status manually nor making any complex logic to enable or disable the continue button.
```
Comment (hrodrick) [inline docs/pages/kmp-quickstart.md:102]: Perhaps we can change this phrase to something like 
```suggestion
And that's one of the easiest forms we can create using **FlowForms** in KMP, we don't need to care about managing the field's status manually nor making any complex logic to enable or disable a continue or save changes button.
```

**Review by hrodrick — COMMENTED** [review]
Nice work! Just left some comments

File: docs/pages/ios-quickstart.md
Comment (hrodrick) [inline docs/pages/ios-quickstart.md:1]: I forget to mention, please include a link to this page in the `Excellent! Lets get started` section of the repo's Readme.md

**Review by hrodrick — APPROVED** [review]

**Pull request merged**